Allows to build vidix output with GCC4 compiler that is stricter than 3.x
versions.

Being an external library, xine-devel upstream does not accept this patch.
It was reported to vidix upstream though.

http://sf.net/tracker/?func=detail&aid=1247940&group_id=76356&atid=546858
diff -x '*~' -ur xine-lib-1.0.1/src/video_out/libdha/sysdep/AsmMacros_generic.h xine-lib-1.0.1-gcc4/src/video_out/libdha/sysdep/AsmMacros_generic.h
--- xine-lib-1.0.1/src/video_out/libdha/sysdep/AsmMacros_generic.h	2005-04-26 10:09:11.000000000 +0200
+++ xine-lib-1.0.1-gcc4/src/video_out/libdha/sysdep/AsmMacros_generic.h	2005-05-01 17:38:54.524010720 +0200
@@ -5,7 +5,7 @@
 #ifndef __ASM_MACROS_GENERIC_H
 #define __ASM_MACROS_GENERIC_H
 
-#warn This stuff is not ported on yur system
+#warning This stuff is not ported on yur system
 
 static __inline__ void outb(short port,char val)
 {
diff -x '*~' -ur xine-lib-1.0.1/src/video_out/libdha/sysdep/pci_generic_cpu.c xine-lib-1.0.1-gcc4/src/video_out/libdha/sysdep/pci_generic_cpu.c
--- xine-lib-1.0.1/src/video_out/libdha/sysdep/pci_generic_cpu.c	2005-04-26 10:09:11.000000000 +0200
+++ xine-lib-1.0.1-gcc4/src/video_out/libdha/sysdep/pci_generic_cpu.c	2005-05-01 17:25:01.756610608 +0200
@@ -1,7 +1,7 @@
 /*
     Generic stuff to compile VIDIX only on any system (SCRATCH)
 */
-#warn This stuff is not ported on yur system
+#warning This stuff is not ported on yur system
 
 static int pci_config_type( void )
 {
